### PR TITLE
My home work

### PR DESCRIPTION
**Pull request by mosessaleh**

### CODE REVIEW

File: week0/homework/index.js [hunk 1]
```diff
@@ -1,17 +1,16 @@
 const Person = require("./Person.js");
-
 // remember to write a constructor function
 let jimmy = new Person("Jimmy", 28);
-
 jimmy.addPhone("55551234");
 
-jimmy.call();
+
```
Comment (pmcalabrese) [inline week0/homework/index.js:6]: Why extra line?

**Review by pmcalabrese — APPROVED** [review]

**Comment by mosessaleh** [comment]
??

File: week0/homework/Person.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    this.call = function() {
+        var phone = arr.map(function(o){return o.Phone});
+        if (phone != "") {
+            return "Calling " + this.name + " at " + arr[0]["Phone"] + " ..."; 
+        } else { 
+            return this.name + " has no phone number saved.";
+        }
+    }
+    this.birthday = function() {
+        return "Wishing " + this.name + " a happy 29th birthday!";
```
Comment (r0hitsharma) [inline week0/homework/Person.js:18]: It wasn't clear in the homework, but you should've incremented the age here (Jimmy was 28 in the example), and not hard-coded the display for birthday. For example see: https://github.com/HackYourFuture-CPH/Node.js/pull/3/commits/e69bffdf5a976195ab6cb0cf7e0c9ce67ef37a72 (I gave the same feedback there previously).

**Comment by r0hitsharma** [comment]
Worked well, just left a small comment.

**Pull request closed without merging**